v3.2/usage/ipv6.md

```diff
@@ -112,7 +112,8 @@ steps below.
 If you wish to only use IPv6 (by disabling IPv4) or your hosts only have
 IPv6 addresses, you must disable autodetection of IPv4 by setting `IP`
 to `none`.  With that set you must also pass a `CALICO_ROUTER_ID` to each
-calico-node pod.
+calico-node pod. Also you must add an env `FELIX_HEALTHHOST` to `localhost6`
```

Comment:
It will be better to spell out "environment variable" rather than just "env", if you can. 

Also, on line 116, you have to choose between the possessive and the article, can't have both. I would suggest, instead of  "because the calico-node's health check" to lose the possessive and instead say "because the calico-node health check".

In addition, you have a mismatch between the verb and the noun in the sentence. This is easily solved by adding an "s" to the end of "use", i.e., "uses".

Reply:
ok ,thank you!

Reply:
@emanic